CODEOWNERS

```diff
@@ -3,3 +3,4 @@
 /packages/babel/ @Andarist
 /packages/image/ @tjenkinson
 /packages/node-resolve/ @tjenkinson
+/packages/terser/ @tada5hi
```

Comment:
Can @tada5hi get write access to the repo so that this is no longer a problem to add, or should this be removed from the PR?

Reply:
@jonkoops i think that should be fine. @shellscape told me to add this line.